lib/chef/knife/kvm_vm_create.rb

```diff
@@ -158,58 +171,62 @@ def create_vm
         command = 'uuidgen'
         uuid = run_remote_command(command)
 
+        # It's got to be one or the other, and we've already checked to make sure
+        # both are not null
+        install_source = if config[:location].nil? 
```

Comment:
Might just be preference, but why no use a ternary for simple conditions like this
`install_source = config[:location].nil? ? config[:iso_image] : config[:location]`